platformcomponents/desktop/content-card-item.json

```diff
@@ -0,0 +1,20 @@
+{
+  "content-card-item": {
+    "comment": "Applied to file attachments from Content tab",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=49650%3A108097",
+    "border": "@theme-outline-secondary-normal",
+    "iconForeground": "@theme-common-text-white",
+    "iconBackground": {
+      "default": "@theme-background-accent-normal"
+    },
+    "overlay": "@theme-common-overlay-primary",
```

Comment:
group the entire element under "overlay":
```
"overlay":{
     "background": "@theme-common-overlay-primary".
     "label": {
             "primary": "@theme-text-primary-disabled",
             "secondary": "@theme-text-secondary-disabled"
     }
}
```

Reply:
Pushed commit
`Grouped elements under overlay in content-card-item`
Changing this conversation to Resolved.